### ISSUE TITLE
Missing metrics after first-time-install

### ISSUE THREAD

**Issue by dxniel7402** [IBM/apiconnect-trawler#74]
Simply put, my problem is that I am not seeing the "promised" metrics in Prometheus that are linked here.

I have apiconnect-trawler installed in a cluster on which API Connect is installed in a namespace in my "monitoring" namespace. After some initial problems, it has been running for a week now. Now I wanted to use metrics like apiconnect_health_status or datapower_gateway_peering_primary_info.

API Connect:
![image](https://github.com/IBM/apiconnect-trawler/assets/61451617/21b2bcb2-9acf-41a3-b396-05b90308cc4a)

Manager:
![image](https://github.com/IBM/apiconnect-trawler/assets/61451617/f2631fc2-183b-433d-9a30-42e971417970)

DataPower: too much, more than documented ...

Analytics:
![image](https://github.com/IBM/apiconnect-trawler/assets/61451617/f5489680-bbb0-4eb5-ac18-020a83349b96)

Cert monitoring: Is working as aspected.

Is it possible that I have a configuration error? Have the metrics simply been implemented yet? 

```yaml
trawler:
  frequency: 10
  use_kubeconfig: false
logging: 
  level: debug
  format: json
prometheus:
  port: 63512
  enabled: true
graphite:
  enabled: false
nets:
  datapower:
    enabled: true
    timeout: 5
    username: admin
    namespace: apic
  manager:
    enabled: true
    username: admin
    namespace: apic
    max_frequency: 300
    process_org_metrics: true
    grant_type: password
  analytics:
    enabled: true
    namespace: apic
  certs:
    enabled: true
  product:
    enabled: true
    username: admin
    namespace: apic
```

**Comment by rickymoorhouse** [comment]
Thanks for the bug report - seems there are two key issues:
 -  whilst the gateway_peering status code was added, it's not being called in the main loop 
 - I have a mistake in the config sample in the docs - where it says `product` it should say `apiconnect`

**Comment by rickymoorhouse** [comment]
The gateway peering function fix is now in and v1.8.0 should be shortly available containing that and the custom analytics intervals and I've corrected the README as well 

Thanks
